src/components/Authentication/Signup/Signup.jsx

```diff
@@ -0,0 +1,141 @@
+import loginSignupImage from "../../../assets/login-signup.png";
+import "@fontsource/dm-sans";
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import { useNavigate } from "react-router-dom";
+
+function Signup() {
+  const navigate = useNavigate();
+
+  const validationSchema = Yup.object().shape({
+    name: Yup.string()
+      .min(3, "Name must be at least 3 characters")
+      .matches(/^[a-zA-Z]+$/, "Name must contain only letters"),
+    email: Yup.string()
+      .email("Invalid email format")
+      .required("Email is required"),
+    password: Yup.string()
+      .required("Password is required")
+      .matches(
+        /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]{8,}$/,
+        "Invalid password."
+      ),
+  });
+
+  const formik = useFormik({
+    initialValues: {
+      name: "",
+      email: "",
+      password: "",
+    },
+    validationSchema: validationSchema,
+    onSubmit: () => {
+      if (
+        !formik.errors.name &&
+        !formik.errors.email &&
+        !formik.errors.password
+      ) {
+        navigate("/login");
+      }
+    },
+  });
+
+  return (
+    <div className="signup flex h-screen font-sans">
```

Comment:
why use the class name "signup" ? Do you do something with it ? 
I see there are multiple places like this, if you don't need them, can you remove all of them ?

Reply:
Removed the class name for "signup" and the others.